src/appleseed/renderer/kernel/rendering/final/adaptivepixelrenderer.cpp

```diff
@@ -297,20 +297,18 @@ namespace
 
                 if (m_sample_aov_tile)
                 {
-                    value[0] = static_cast<float>(trackers[0].get_size());
+                    value[0] += static_cast<float>(trackers[0].get_size());
 
                     m_sample_aov_tile->set_pixel(pt.x, pt.y, value);
                 }
 
                 if (m_variation_aov_tile)
                 {
-                    value[0] =
-                        saturate(
-                            max(
-                                trackers[0].get_variation(),
-                                trackers[1].get_variation(),
-                                trackers[2].get_variation())
-                            / m_params.m_max_variation);
+                    value[0] +=
```

Comment:
And this one?

Reply:
This one is less clear since you're no longer dividing by `m_params.m_max_variation`.

Reply:
Indeed. I should divide by `m_params.m_max_variation * m_params.m_passes`.